src/ui/wxWidgets/YubiMixin.cpp

```diff
@@ -81,6 +84,25 @@ void YubiMixin::HandlePollingTimer()
   }
 }
 
+void YubiMixin::SetPollingInterval(int value) {
+  if (value <= 0) {
+    s_pollingInterval = YubiMixin::POLLING_INTERVAL; // not really relevant in that case, but just reset to default
+    s_pollingEnabled = false;
+  }
+  else if (value < YubiMixin::POLLING_INTERVAL_MIN /* ms */) {
+    s_pollingInterval = YubiMixin::POLLING_INTERVAL_MIN; // limit the polling interval to a minimum
+    s_pollingEnabled = true;
+  }
+  else if (value > YubiMixin::POLLING_INTERVAL_MAX /* ms */) {
+    s_pollingInterval = YubiMixin::POLLING_INTERVAL_MAX; // limit the polling interval to a maximum
+    s_pollingEnabled = true;
+  }
+  else {
+    s_pollingInterval = value;
+    s_pollingEnabled = true;
```

Comment:
Nit: if the only case when polling is disabled is the < 0 case, why not move this repeating line out of the if branches? E.g.
```c++
  s_pollingEnabled = true;
  if (value <= 0) {
    s_pollingInterval = YubiMixin::POLLING_INTERVAL;
    s_pollingEnabled = false;
  }
```

Reply:
Changed as suggested.